protected_environments.go

```diff
@@ -114,8 +115,9 @@ func (s *ProtectedEnvironmentsService) GetProtectedEnvironment(pid interface{},
 // GitLab API docs:
 // https://docs.gitlab.com/ee/api/protected_environments.html#protect-repository-environments
 type ProtectRepositoryEnvironmentsOptions struct {
-	Name               *string                      `url:"name,omitempty" json:"name,omitempty"`
-	DeployAccessLevels *[]*EnvironmentAccessOptions `url:"deploy_access_levels,omitempty" json:"deploy_access_levels,omitempty"`
+	Name                  *string                      `url:"name,omitempty" json:"name,omitempty"`
+	RequiredApprovalCount *int                         `url:"required_approval_count,omitempty" json:"required_approval_count,omitempty"`
+	DeployAccessLevels    *[]*EnvironmentAccessOptions `url:"deploy_access_levels,omitempty" json:"deploy_access_levels,omitempty"`
```

Comment:
In the docs these are ordered the other way around, so `RequiredApprovalCount` comes after `DeployAccessLevels`...

Reply:
Corrected